websites/S/Steam/presence.ts

```diff
@@ -23,314 +19,471 @@ presence.on("UpdateData", async () => {
 
 		if (document.location.pathname === "/" || !document.location.pathname) {
 			if (subsection) {
-				presenceData.state = `Browsing ${subsection}.`;
-
+				presenceData.state = `Browsing ${subsection}`;
 				presenceData.startTimestamp = browsingTimestamp;
 			} else {
-				presenceData.state = "Home";
-
+				presenceData.state = "Homepage";
 				presenceData.startTimestamp = browsingTimestamp;
 			}
+		} else if (document.location.pathname.includes("/filedetails/")) {
+			presenceData.details = "Viewing a community post";
+			presenceData.state = document
+				.querySelector<HTMLTitleElement>("head > title")
+				.textContent.replace("Steam Community :: ", " ")
+				.replace("::", "|");
+			presenceData.startTimestamp = browsingTimestamp;
+			presenceData.buttons = [
+				{
+					label: "View the page",
+					url: document.location.href,
+				},
+			];
+		} else if (
+			document.location.pathname.includes("/id") ||
+			document.location.pathname.includes("/profiles")
```

Comment:
Properties of `document.location` are used often. To simply the code, it may be a good idea to destructure them. 

`const {pathname, href, hostname} = document.location;`

Reply:
Thanks for the explanation
I've done it 👍

Reply:
Ensure that you mark conversations you've completed as resolved.